### ISSUE TITLE
draw_functions float2

### ISSUE THREAD

**Issue by Jesus05** [imp5imp5/dasbox#5]
Очень хочется альясы для всяких set_pixel\line\reсt с float2 в качестве координаты.

**Comment by imp5imp5** [comment]
Функции с float2 будут добавлены после перехода с SFML на BGFX

**Comment by spiiin** [comment]
@imp5imp5 а можно будет при переезда на bgfx еще sfml на sdl заменить? хочется под web и мобильный платформы собрать попробовать

**Comment by imp5imp5** [comment]
@spiiin bgfx может работать под web: https://bkaradzic.github.io/bgfx/examples.html#metaballs
В поддерживаемых платформах также заявлено: Android (14+, ARM, x86, MIPS), iOS (iPhone, iPad, AppleTV)

**Comment by spiiin** [comment]
Так, но выбор бекэнда то на стороне клиентского кода? https://github.com/bkaradzic/bgfx/tree/master/examples/common/entry
вот я тут прошу выбрать тот, который без изменений почти должен по разные платформы собраться, чтобы не реимплементить эту часть

**Comment by imp5imp5** [comment]
@spiiin да, конечно, но также будет оставлен доступ к функциям bgfx
